cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -829,6 +832,44 @@ public KafkaBrokerConfigurationBuilder withLogDirs(List<VolumeMount> mounts)  {
 
         return this;
     }
+    /**
+     * Configure the tiered storage configuration for Kafka brokers.
+     *
+     * @param clusterName     Name of the cluster
+     * @param tieredStorage   TieredStorage configuration.
+     *
+     * @return  Returns the builder instance
+     */
+    public KafkaBrokerConfigurationBuilder withTieredStorage(String clusterName, TieredStorage tieredStorage)  {
+        if (tieredStorage == null) {
+            return this;
+        }
+        printSectionHeader("Kafka tiered storage configuration");
+        writer.println("# RLMM configuration generated by Strimzi");
+        writer.println("remote.log.storage.system.enable=true");
+        writer.println("remote.log.metadata.manager.impl.prefix=rlmm.config.");
+        writer.println("remote.log.metadata.manager.class.name=org.apache.kafka.server.log.remote.metadata.storage.TopicBasedRemoteLogMetadataManager");
+        writer.println("remote.log.metadata.manager.listener.name=REPLICATION-9091");
+        writer.println("rlmm.config.remote.log.metadata.common.client.bootstrap.servers=" + clusterName + "-kafka-brokers:9091");
+        writer.println("rlmm.config.remote.log.metadata.common.client.security.protocol=SSL");
+        writer.println("rlmm.config.remote.log.metadata.common.client.ssl.keystore.location=/tmp/kafka/cluster.keystore.p12");
+        writer.println("rlmm.config.remote.log.metadata.common.client.ssl.keystore.password=${CERTS_STORE_PASSWORD}");
+        writer.println("rlmm.config.remote.log.metadata.common.client.ssl.keystore.type=PKCS12");
+        writer.println("rlmm.config.remote.log.metadata.common.client.ssl.truststore.location=/tmp/kafka/cluster.truststore.p12");
+        writer.println("rlmm.config.remote.log.metadata.common.client.ssl.keystore.password=${CERTS_STORE_PASSWORD}");
```

Comment:
This is typo I guess?

```suggestion
        writer.println("rlmm.config.remote.log.metadata.common.client.ssl.truststore.password=${CERTS_STORE_PASSWORD}");
```